packages/mobile/src/home/NotificationBox.tsx

```diff
@@ -246,7 +246,7 @@ export class NotificationBox extends React.Component<Props, State> {
                 notificationType: NotificationBannerTypes.invite_prompt,
                 selectedAction: NotificationBannerCTATypes.accept,
               })
-              navigate(Screens.Invite)
+              // TODO: navigate to relevant invite flow
```

Comment:
Shouldn't we also delete this notification?

Reply:
I opted to leave it in with a to-do because I thought we will probably have a new invite flow sometime soon and we'll want a notification for it. If that's not the case, we can remove it